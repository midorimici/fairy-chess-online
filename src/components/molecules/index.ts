export { BreadcrumbNav } from './BreadcrumbNav';
export { Container } from './Container';
export { GameList } from './GameList';
