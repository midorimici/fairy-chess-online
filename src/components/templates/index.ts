export { Game } from './Game';
export { Games } from './Games';
export { Home } from './Home';
export { NewRoom } from './NewRoom';
