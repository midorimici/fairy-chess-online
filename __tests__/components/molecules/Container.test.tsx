import { render } from '@testing-library/react';
import { Container } from '~/components/molecules';

it('renders Container unchanged', () => {
  const { asFragment } = render(<Container>Main contents</Container>);
  expect(asFragment()).toMatchSnapshot();
});
