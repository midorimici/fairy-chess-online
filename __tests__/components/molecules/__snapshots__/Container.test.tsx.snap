// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Container unchanged 1`] = `
<DocumentFragment>
  <div
    class="chakra-container css-1y4v3tz"
  >
    <main
      class="css-i9gxme"
    >
      Main contents
    </main>
    <div
      class="css-gmuwbf"
    >
      Â© 2022-22 midorimici
    </div>
  </div>
</DocumentFragment>
`;
